### PR TITLE
Exporting worker init function

### PR DESCRIPTION
**Pull request by silimarius**
- Exporting worker init function
- Updated TS config for deployment and test
- Updated peer dependencies
- Updated build
- Added installation and usage instructions in README

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -87,10 +86,10 @@
     "zustand": "^4.3.9"
   },
   "peerDependencies": {
-    "react": ">=18.12.0",
-    "react-dom": ">=18.12.0",
-    "snarkyjs": ">=0.12.1",
-    "zod": ">=3.21.4",
+    "react": ">=18.0.0",
+    "react-dom": ">=18.0.0",
+    "snarkyjs": ">=0.11.0",
```
Comment (Mautjee) [inline package.json:91]: Why are we not using the latest version?

File: src/zkAppWorker.ts [hunk 2]
```diff
@@ -98,15 +98,15 @@ const workerFunctions = {
       state.tree.getWitness(state.transitionIndex),
     );
 
-    console.info("creating init proof...");
+    console.info("[zk-states worker] creating init proof...");
```
Comment (Mautjee) [inline src/zkAppWorker.ts:101]: Do we want to keep these console logs in prod version?

File: package.json (hunk 1 above)
Comment (silimarius) [reply package.json:91]: peer dependencies are what the user should install in their project, not what we use to build. here we are saying that they should have at least those versions. wht we use to build is specified in dev deps

File: src/zkAppWorker.ts (hunk 2 above)
Comment (silimarius) [reply src/zkAppWorker.ts:101]: for now yes, in the future we might consider using a more advanced logging lib

**Comment by Mautjee** [comment]
LGTM

**Review by Mautjee — APPROVED** [review]

**Pull request merged**